### ISSUE TITLE
UI: Impossible to reap account with 1.0069DOT

### ISSUE THREAD

**Issue by rvalle** [polkadot-js/apps#10447]
* **I'm submitting a ...**

  - [x] Bug report
  - [ ] Feature request
  - [ ] Support request
  - [ ] Other

* **What is the current behavior and expected behavior?**

When attempting to transfer and reap account with this kind of edge balance, a warning is displayed before having opportunity to select transfer all balance and reap account in the UI.

The expected behavior: you should be able to access UI controls required to transfer and reap this account balance.

* **What is the motivation for changing the behavior?**

The current implementation is broken:

The warning "the transaction, after application of fees, will drop..." needs to disappear when "transfer with account keep-alive checks" is disabled, currently it does not and the UI to "transfer all balance and reap..." is never displayed.

**Comment by TarikGul** [comment]
Thanks for the issue. It seems sensible, but can you provide an image of the current UI, and functionality that you are referencing?

**Comment by rvalle** [comment]
![Screenshot from 2024-04-11 08-04-40](https://github.com/polkadot-js/apps/assets/412837/dfcb8847-a24c-433e-8896-3b8290e8b284)

You see, I just disabled keep alive, how am I supposed to select reap account if the warning sign blocks it?

**Comment by ap211unitech** [comment]
@rvalle  After some investigation, It turned out that this issue appears when transferrable amount (after deducting fee) is less than existential amount. For example, let's take Paseo relay chain. It requires 1 PAS as existential amount. 

Now, I have an account with balance 1.01 PAS. I essentially can not transfer this amount to any account (doesn't matter if we want it to be reaped or not) because it takes 0.0174135772 PAS as fee. 
The net transferrable balance would be = 1.01 - 0.0174135772 = 0.9925864228 PAS which goes below the required existential amount. 

Even if we don't use the Apps UI and try to do it with some script or with any other application, it still won’t work because this is fundamentally how the blockchain operates in Polkadot Ecosystem. Also, even if you use any other extrinsic to transact on blockchain, it won't allow you to do so if the sender's account balance goes below the existential deposit as a result. 

```ts
import { ApiPromise, Keyring, WsProvider } from "@polkadot/api";
import type { KeyringPair$Json } from "@polkadot/keyring/types";

import fs from "node:fs";

const keyring = new Keyring({ type: "sr25519" });

const main = async () => {
  const api = await ApiPromise.create({
    provider: new WsProvider("wss://paseo-rpc.dwellir.com"),
  });

  await api.isReadyOrError;

  const account = "5HpCBJf1Erd1ZWsswFRZfdVXsVrC418WoaoGVkz1H9f7mELF";

  const senderKeyring = keyring.addFromJson(
    JSON.parse(fs.readFileSync(`${account}.json`, "utf-8")) as KeyringPair$Json
  );

  senderKeyring.unlock("<your password>");

  // Reap Account
  const reap_ext = api.tx.balances.transferAll(
    "15QW9wnU2o3jQsqNqFHQqFQyWNgi1epMAkXEkBbheFBpHYW3",
    false
  );
  await reap_ext.signAndSend(senderKeyring);

  await api.disconnect();
};

main()
  .catch((e) => console.log("Something went wrong!", e))
  .finally(() => process.exit(0));
```

Used this script as to alter Apps UI, but end up getting the `Invalid Transaction: Inability to pay some fees , e.g. account balance too low` error.

For me, It works fine when I increased my balance to 1.02 PAS as my balance after transaction would be more than existential deposit (1 PAS).

### Basically, this is the reason why you cannot alter that warning message and cannot reap the account at this stage.
